data/FilterList.json

```diff
@@ -2484,7 +2575,7 @@
     "homeUrl": "https://www.facebook.com/adblockpl",
     "issuesUrl": "https://github.com/adblockpolska/Adblock_PL_List/issues",
     "licenseId": 4,
-    "name": "Adblock Polska",
+    "name": "Adblock Polska (Obsolete)",
```

Comment:
I think we should avoid adding non-title information to the `name` fields as much as possible. Obsolete/Unsupported/etc. duplicates the same info that `updatedDate` notes. We could add a business rule to display something like "Obsolete" in the UI based on the value of `updatedDate` if there was demand for that. For now, let's avoid adding these descriptors to `name` and have any further discussion over in #566